test/webgl/framebuffer.spec.js

```diff
@@ -183,7 +183,7 @@ function testFramebufferReadPixels(t, gl) {
   const clearColor = [1, 0.5, 0.25, 0.125];
   const expectedColor = [255, 128, 64, 32];
 
-  clear(gl, {framebuffer, color: clearColor});
+  framebuffer.clear({color: clearColor});
```

Comment:
This unit test is testing both, `clear` and `readPixels`, no new unit test needed.

Reply:
> This unit test is testing both, clear and readPixels, no new unit test needed.

One could argue that for clarity and completeness, it would be good to have a "named test case" for each Framebuffer method. While both method calls are indeed being exercised here, they are inside a big test case that focuses on another thing, and it is not clear to me that future maintainers will understand that these calls must be kept here lest we lose coverage.

Nit: Is there still a separate test for the global function `clear`, or was that lost in this change?

Reply:
I changed `clear` to `framebuffer.clear` in this unit test, but `clear` is being tested also given it is called from `framebuffer.clear`, we should also have `clear-spec.js`.

Reply:
> I changed clear to framebuffer.clear in this unit test, but clear is being tested also given it is called from framebuffer.clear, we should also have clear-spec.js.

Exactly, the implementation of `framebuffer.clear` could change, so we shouldn't rely on that for coverage. 

We still have a lot of untested functions, and there is no way we can address it all in a single PR, but your current approach of adding a little bit more coverage than we need in each PR really helps narrow the gap!